x-pack/plugins/apm/public/services/rest/apm.js

```diff
@@ -64,6 +64,17 @@ export async function loadServiceDetails({ serviceName, start, end, kuery }) {
   });
 }
 
+export async function loadTraceList({ start, end, kuery }) {
+  return callApi({
+    pathname: '/api/apm/traces',
+    query: {
+      start,
+      end,
+      esFilterQuery: await getEncodedEsQuery(kuery)
+    }
+  });
+}
+
```

Comment:
Microscopic nit:
It just occurred to me that it might make sense to group the methods together by endpoints, eg. `loadTraceList` and `loadTrace`. Either in a separate file like `rest/apm/traces.ts` (to mirror the server), or just closer together in this file.

Reply:
Waiting on re-ordering these for now, but I agree with you. Can clean these up later when we're not both in the files at the same time.

Reply:
totally fine. Not necessary in this PR.